app/views/users/webauthn_setup/new.html.erb

```diff
@@ -1,14 +1,16 @@
+<% setup_context = @platform_authenticator ? 'webauthn_platform_setup' : 'webauthn_setup' %>
+
 <% title t('titles.totp_setup.new') %>
 <% help_link = new_window_link_to t('links.what_is_webauthn'),
                                   MarketingSite.help_hardware_security_key_url %>
 
 <%= image_tag asset_url('security-key.svg'), alt: '', width: '90', class: 'margin-left-1' %>
 
 <h1 class='margin-y-0'>
-  <%= t('headings.webauthn_setup.new') %>
+  <%= t("headings.#{setup_context}.new") %>
```

Comment:
```suggestion
  <%= t('headings.#{setup_context}.new') %>
```

Reply:
Agree that these should be double quotes, but let's plan on doing that for ALL `erb` templates in the future puhleez.

Reply:
I'm pretty sure there is a linter rule that requires single quotes for all strings unless there is interpolation or you have an escaped character.

Reply:
ERBLint checks for this, but we have a lot of problem files that existed before the linter was added which are [exempted](https://github.com/18F/identity-idp/blob/521b4386e673ad6a2e9b1e49cc4f01f0a40111ce/.erb-lint.yml#L5-L105) from enforcement (including this one), but are [slowly chipping away at them](https://github.com/18F/identity-idp/commits/main/.erb-lint.yml) over time.